resources/careconnect-to-ukcore/allergyintolerance/expected/AllergyIntoleranceDescriptionId-Extension-3to4_000.json

```diff
@@ -0,0 +1,33 @@
+{
+  "resourceType": "AllergyIntolerance",
+  "code": {
+    "coding": [
+      {
+        "system": "http://read.info/readv2",
+        "code": "TJ03000",
+        "display": "Adverse reaction to erythromycin",
+        "userSelected": true
+      },
+      {
+        "extension": [
+          {
+            "url": "https://fhir.hl7.org.uk/StructureDefinition/Extension-UKCore-CodingSCTDescId",
+            "extension": [
+              {
+                "url": "descriptionId",
+                "valueId": "2579700012"
```

Comment:
this should be a valueIdentifier, i.e.

                 "valueIdentifier": {
                   "system": "http://snomed.info/sct",
                   "value": "2579700012"
                 }